resources/src/androidLibMain/kotlin/StyledString.kt

```diff
@@ -54,7 +104,16 @@ actual class StyledStringBuilder constructor(
 
     private val builder = SpannableString(string)
 
+    /**
+     * Adds a [StringStyleAttribute] for a given range
+     * @param attribute the [StringStyleAttribute] to apply
+     * @param range the [IntRange] at which to apply the style
+     * @throws [IndexOutOfBoundsException] if [range] is out of bounds for the text to span
+     */
     actual fun addStyleAttribute(attribute: StringStyleAttribute, range: IntRange) {
+        if (range.first < 0 || range.last >= builder.length) {
```

Comment:
Can be `range !in 0 until builder.length`

Reply:
Doesnt work, changed it to `if (range.all { it in builder.indices })`